seirmo/tests/test_models.py

```diff
@@ -51,6 +53,13 @@ def test_simulate(self):
         expected = np.ones(shape=n_times)
```

Comment:
could you create a separate test method that checks behaviour with the reutrn_incidence = True